_layouts/post.liquid

```diff
@@ -18,8 +18,9 @@ layout: default
   <header class="post-header">
     <h1 class="post-title">{{ page.title }}</h1>
     <p class="post-meta">
-      {{ page.date | date: '%B %d, %Y' }}
+      Created: {{ page.date | date: '%B %d, %Y' }}
       {% if page.author %}• {{ page.author }}{% endif %}
+      {% if page.last_updated %} &nbsp; | &nbsp; Last Updated: {{ page.last_updated | date: '%B %d, %Y' }}{% endif %}
```

Comment:
Here I think it would be better as `, last updated in` instead of ` &nbsp; | &nbsp; Last Updated:`

Reply:
I have modified the code according to your suggestions.